readme.txt

```diff
@@ -21,5 +21,15 @@ Full documentation on how to configure and integrate the plugin is available wit
 Detailed instructions on how to install the plugin is available within the [GitHub repository](https://github.com/passle/passle-sync-wordpress).
 
 == Changelog ==
+
+= 1.1.0 =
+* Added health check page, allowing you to perform automated checks to ensure the integration is set up correctly.
+* Restructured webhooks to allow for more flexibility in the future. This is a breaking change, and auto-sync will stop working on previous versions of the plugin.
+
 = 1.0.0 =
-* Initial release
\ No newline at end of file
+* Initial release
+
+== Upgrade Notice ==
+
+= 1.1.0 =
+Restructured webhooks to allow for more flexibility in the future. This is a breaking change, and auto-sync will stop working on previous versions of the plugin.
```

Comment:
I'm not familiar with the extra format/syntax used, but should there be a `*` at the start of this line?

Reply:
The changelog is a bulleted list, but this is just a single notice giving a reason that the user should upgrade to version 1.1.0, so there is no need for a `*`.